### PR TITLE
php classes

### PR DESCRIPTION
**Pull request by franzholz**
Releases: master, 10.4

### CODE REVIEW

File: Documentation/ColumnsConfig/Type/Passthrough/Index.rst [hunk 1]
```diff
@@ -21,26 +21,27 @@ These are the :ref:`none type <columns-none>`, the :ref:`passthrough type <colum
 
 Characteristics of `passthrough`:
 
-* A value sent to the DataHandler is just kept as is and put into the database field. Default FormEngine
+* A value sent to the
```
Comment (DavidBruchmann) [inline Documentation/ColumnsConfig/Type/Passthrough/Index.rst:24]: DataHandler is missing now?

File: Documentation/ColumnsConfig/Type/Passthrough/Index.rst [hunk 2]
```diff
@@ -21,26 +21,27 @@ These are the :ref:`none type <columns-none>`, the :ref:`passthrough type <colum
 
 Characteristics of `passthrough`:
 
-* A value sent to the DataHandler is just kept as is and put into the database field. Default FormEngine
+* A value sent to the `DataHandler`
+  is just kept as is and put into the database field. Default `FormEngine`
   however never sends values.
```
Comment (mbrodala) [inline Documentation/ColumnsConfig/Type/Passthrough/Index.rst:26]:
```suggestion
* A value sent to the :php:`DataHandler` is kept as is and put into the database field. However, the default TYPO3 backend forms never send data for a `passthrough` field.
```

File: Documentation/ColumnsConfig/Type/Passthrough/Index.rst [hunk 3]
```diff
@@ -21,26 +21,27 @@ These are the :ref:`none type <columns-none>`, the :ref:`passthrough type <colum
...
 * Unlike none, type passthrough must have a database field.
```
Comment (mbrodala) [inline Documentation/ColumnsConfig/Type/Passthrough/Index.rst:27]:
```suggestion
* Unlike the field type `none`, the field type `passthrough` must have a database field.
```

File: Documentation/ColumnsConfig/Type/Passthrough/Index.rst [hunk 4]
```diff
@@ -21,26 +21,27 @@ These are the :ref:`none type <columns-none>`, the :ref:`passthrough type <colum
...
-* FormEngine does not render anything for passthrough types by default. But it can be combined with a custom
+* `FormEngine` does not render anything for passthrough types by default. But it can be combined with a custom
```
Comment (mbrodala) [inline Documentation/ColumnsConfig/Type/Passthrough/Index.rst:28]:
```suggestion
* The TYPO3 backend forms do not render anything for `passthrough` fields by default. But they can be combined with a custom
```

File: Documentation/ColumnsConfig/Type/Passthrough/Index.rst [hunk 5]
```diff
@@ -21,26 +21,27 @@ These are the :ref:`none type <columns-none>`, the :ref:`passthrough type <colum
...
   renderType to make it render something. A user type is better suited for such use cases, though.
```
Comment (mbrodala) [inline Documentation/ColumnsConfig/Type/Passthrough/Index.rst:29]:
```suggestion
  `renderType` to display something. A field of type `user` is better suited for such use cases, though.
```

File: Documentation/ColumnsConfig/Type/Passthrough/Index.rst [hunk 6]
```diff
@@ -21,26 +21,27 @@ These are the :ref:`none type <columns-none>`, the :ref:`passthrough type <colum
...
-* Type passthrough field values are usually also not rendered at other places in the backend.
-* Field updates by the DataHandler get logged and the history/undo function will work with such values.
+* Type passthrough field values are usually not rendered at other places in the backend.
```
Comment (mbrodala) [inline Documentation/ColumnsConfig/Type/Passthrough/Index.rst:30]:
```suggestion
* Values of `passthrough` fields are usually not rendered at other places in the backend.
```

File: Documentation/ColumnsConfig/Type/Passthrough/Index.rst [hunk 7]
```diff
@@ -21,26 +21,27 @@ These are the :ref:`none type <columns-none>`, the :ref:`passthrough type <colum
...
+* A field needs no rendering, but some data handling using hooks of the `DataHandler`.
 * The passthrough type is used by TYPO3 core to handle meta data on records rows that are not shown as fields
   if editing records and only have data logic attached to it. An example is the `l18n_diffsource` field whose
   data is rendered differently in FormEngine at other places if editing a record but still updated and handled
-  by the DataHandler.
+  by the `DataHandler`.
 
-Typical usages of type passthrough is a field that only needs data evaluation on the DataHandler side, but
-no rendering definition. The DataHandler does not evaluate the value in any way by default.
+Typical usages of type passthrough is a field that only needs data evaluation on the `DataHandler` side, but
```
Comment (mbrodala) [inline Documentation/ColumnsConfig/Type/Passthrough/Index.rst:41]:
```suggestion
Typical usages of the field type `passthrough` is a field that only needs data evaluation on the `DataHandler` side, but
```

**Review by mbrodala — CHANGES_REQUESTED** [review]

**Review by mbrodala — APPROVED** [review]

**Pull request merged**